VideoLocker/src/main/java/org/edx/mobile/view/CourseOutlineActivity.java

```diff
@@ -50,6 +50,10 @@ protected void onLoadData() {
                 courseData.getCourse().getId(), courseComponentId);
         setTitle(courseComponent.getName());
 
+        // onLoadData is only called in case of viewing a section
```

Comment:
Yeah, it is being called from `CourseBaseActivity#restore` function incase of Course Outline. Need to work around it somehow.

>Therefore I guess you can remove the duplicate tracking call in onCreate().

Nope, there's a difference between the 2 calls, one is for Course Outline and other is for Section Outline. We need both.